herc/aurorabackend.py

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Personally I'm not a huge fan of having a whole module just for an exception.  I think once we refactor these `aurora*.py` classes into a sub-package this can probably go in either `__init__.py` or perhaps `__init__.py` of the `herc` package... there are lots of options.